GVFS/GVFS.Common/InstallerPreRunChecker.cs

```diff
@@ -147,6 +147,16 @@ protected virtual bool IsProjFSInboxed()
             return GVFSPlatform.Instance.KernelDriver.IsGVFSUpgradeSupported();
         }
 
+        protected virtual bool FreshInstallOrServiceRunning()
+        {
+            bool isInstalled;
+            bool isRunning;
+
+            GVFSPlatform.Instance.IsServiceInstalledAndRunning(GVFSConstants.Service.ServiceName, out isInstalled, out isRunning);

```

Comment:
with C# 7, you should be able to define the bools inside the method:

`GVFSPlatform.Instance.IsServiceInstalledAndRunning(GVFSConstants.Service.ServiceName, out bool isInstalled, out bool isRunning)`